doc/rst/developer/bestPractices/ContributorInfo.rst

```diff
@@ -724,6 +724,55 @@ developing yourself (or code that you've developed as a standalone package),
 alert the chapel-developers_ mailing list of this as, presently, such code
 packages must be approved by Cray leadership before being committed.
 
+Here are some guiding questions to determine whether a third-party package you
+rely on should be committed to the chapel repository:
+
+- How large is the third-party code you wish to include?
+
+  - If the code is very large, perhaps it would be better to add directions on
+    how to install this dependency.
+
+- Under what license does this code operate?
+
+  - We try not to add dependencies on code that is under GPL or LGPL, as those
+    licenses are copyleft and force derivative works to be distributed under the
```

Comment:
This parses awkwardly to me.  "copyleft" is a noun rather than an adjective, isn't it?  "are reciprocal"?  "have copyleft properties"?  (that still seems to use it as an adjective, but reads more smoothly to me).